src/Type/Definition/Type.php

```diff
@@ -267,7 +267,6 @@ public function jsonSerialize(): string
 
     public static function reset(): void
     {
-        static::$standardTypes = null;
```

Comment:
This needs to be reset, in my use case (running phpunit tests that once set this)

Reply:
What if we add a second argument to the overrideStandardTypes method that resets it completely? By default it's off, to mimic the existing logic. But it allows to reset and override fully.

Reply:
Pushed a commit to explain what I mean

Reply:
@spawnia Pushed a commit to explain what I mean